packages/wallet-management/src/wallets.ts

```diff
@@ -255,6 +255,14 @@ const okx: Wallet = new InjectedConnector(
   (window as any).okxwallet,
 );
 
+const rabbywallet: Wallet = new InjectedConnector(
+  {
+    name: 'Rabby Wallet',
+    installed: async () => (window as any).ethereum,
```

Comment:
looked at the wallet injection. for the installed hook
`(window as any).ethereum. isRabby` 
should work

Reply:
actually rabby wallet injects also as `window.rabby` we should use that instead of the window.ethereum injection

Reply:
so: installed just checks window.rabby